CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
     - Added `mainAxisSize`, `mainAxisAlignment`, `rightIconConfig` and `leftIconConfig`.
 - Adds additional configurations for `CalendarPageHeader`, `MonthPageHeader`, `DayPageHeader` and `WeekPageHeader`.
     - Added `titleBuilder` to build custom title for header.
+- Fixes issue calendar scroll physics for day & week view
```

Comment:
Mention the issue number and link in the Release notes. Refer previous release notes.